apps/web/src/components/layout/footer/footer.tsx

```
@@ -42,11 +47,12 @@ const Footer: React.FC = () => {
             <OnRuntimeWordMark height={24} className="h-6" />
 
             <p>
-              Un studio de créatif, à travers le monde.
+              {t("tagline")}
               <br />
-              Mais on peut se voir sur{" "}
-              <strong className="text-foreground">Rouen</strong> et{" "}
-              <strong className="text-foreground">Paris</strong>.
+              <span
+                dangerouslySetInnerHTML={{ __html: t("location") }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)